velox/core/QueryConfig.h

```diff
@@ -135,6 +135,15 @@ class QueryConfig {
 
   static constexpr const char* kTestingSpillPct = "testing.spill-pct";
 
+  /// The max allowed spilling level with zero being the initial spilling level.
+  /// This only applies for hash build spilling which might trigger recursive
+  /// spilling when the build table is too big. If it is set to -1, then there
```

Comment:
Should we provide some guideline on how to set this value? Would the user understand the implicit limit and choose this value to be less than that?